Autopilot/GPS/Makefile

```diff
@@ -0,0 +1,113 @@
+#
```

Comment:
Did not know that. However, keeping the Makefiles does make it quicker to clone & set up the repo again.